query-engine/connector-test-kit-rs/query-test-macros/src/args/connector_test.rs

```diff
@@ -28,8 +28,12 @@ pub struct ConnectorTestArgs {
     #[darling(default)]
     pub capabilities: RunOnlyForCapabilities,
 
+    // #[deprecated(since = "4.5.0", note = "Use `relation_mode` instead")]
     #[darling(default)]
-    pub referential_integrity: Option<ReferentialIntegrity>,
+    pub referential_integrity: Option<RelationMode>,
```

Comment:
For the QE tests, I think we could already remove that, unless it's too big of a diff.

Reply:
Thanks, I'll try that in a later PR.